jazz-ui/src/app/pages/admin-dashboard/admin-dashboard.component.scss

```diff
@@ -0,0 +1,44 @@
+@import './../../../main-styles';
+
+.page-title-wrap {
+  overflow: hidden;
+  padding: 26px 0 26px 0px;
+  .header1 {
+    float: left;
+  }
+  btn-primary-with-icon {
```

Comment:
is this correct?

Reply:
That refers to the component <btn-primary-with-icon> and was taken from the service-detail scss.